tests/data/group_system/group_logging/group_ensure_rsyslog_log_file_configuration/rule_rsyslog_cron_logging/set_cron_logging.sh

```diff
@@ -0,0 +1,13 @@
+# Function to add cron logging line to specific file
+# Expects argument:
+# file:		Configuration file, where we want cron logging set
+#
+function set_cron_logging {
+	local file=$1
+
+	if grep -s '^[[:space:]]*cron\.\*' $file; then
```

Comment:
Shouldn't be the variable quoted?

Reply:
It isn't necessary here, but for perfection it's better with it so I'll add it.